### ISSUE TITLE
Support for rollbar.js 2.x

### ISSUE THREAD

**Issue by websirnik** [tandibar/ng-rollbar#34]
This module is currently behind the official repo: https://github.com/rollbar/rollbar.js/blob/master/CHANGELOG.md

**Comment by pccjamie** [comment]
+1

**Comment by FaberYeah** [comment]
**a lot** behind ... +1

**Comment by azharkhan** [comment]
+1

**Comment by misshula** [comment]
+1

**Comment by amunro** [comment]
+1

**Comment by mpedersen15** [comment]
+1

**Comment by lmadeira** [comment]
+1000 👍

**Comment by mpedersen15** [comment]
Just thought I'd chime in again. I wanted to upgrade this wrapper to use Rollbar v2.2.3 so I could use the new Telemetry feature Rollbar implemented. I forked this repo (check it out [here](https://github.com/mpedersen15/ng-rollbar)) and swapped out the snippet for the v2.2.3 snippet. I tried it out with my own Ionic v1 app and it seemed to work just fine! New Rollbar events have the Telemetry I was looking for. I did not minify the JS at all, just quick and dirty fix for the unminified file.
